packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -16,21 +15,23 @@ class AlertBox extends React.Component {
   }
 
   componentDidUpdate() {
-    if (this.props.isVisible && this.props.scrollOnShow) {
-      this.scrollToAlert();
-    }
+    this.scrollToAlert();
   }
 
-  scrollToAlert() {
+  scrollToAlert = () => {
     const isInView = window.scrollY <= this._ref.offsetTop;
 
+    if (!this.props.isVisible || !this.props.scrollOnShow) {
+      return;
+    }
+
     if (this._ref && !isInView) {
       this._ref.scrollIntoView({
         block: 'end',
         behavior: 'smooth',
       });
     }
-  }
+  };
```

Comment:
Did the linter add this `;`? 🙂

Reply:
it's because it switched from a function declaration to expression I think